### DISCUSSION TITLE
Install all packages from a bucket

### DISCUSSION THREAD

**Discussion by furopi** [ScoopInstaller/Scoop#5259] [General]
Hi guys!

Just a quick question, I added the sysinternals bucket to scoop and now I'd like to install all packages from this bucket at once. Is this possible, and if yes, how?

Help would be gladly appreciated!

Kind regards,
Florian

**Comment by rashil2000** [comment]
The extras bucket has the `sysinternals` package which will install all utilities
`scoop install sysinternals`

**Reply by furopi** [reply]
Thank you very much! <3

**Reply by niheaven** [reply]
Hmm, the full package in sysinternals is sysinternals-suite, which is a fixed version of sysinternals in extras.

**Reply by furopi** [reply]
So what does that mean? Do you have a more suitable suggestion? Thank you! 😊

**Reply by niheaven** [reply]
Since you have add sysinternals bucket, please install sysinternals-suite. I haven't updated the one in `extras`.
